<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParagraphBulletsNumbers</class>
 <widget class="QWidget" name="ParagraphBulletsNumbers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>379</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" rowspan="2">
    <widget class="QListWidget" name="listTypes"/>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="listPropertiesPane" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Display</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Prefix:</string>
           </property>
           <property name="buddy">
            <cstring>prefix</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KLineEdit" name="prefix"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Suffix:</string>
           </property>
           <property name="buddy">
            <cstring>suffix</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="KLineEdit" name="suffix"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="customCharPane" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="customCharLabel">
           <property name="text">
            <string>Custom character:</string>
           </property>
           <property name="buddy">
            <cstring>customCharacter</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="customCharacter">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="countersGroupbox">
        <property name="title">
         <string>Counters format</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>List depth:</string>
           </property>
           <property name="buddy">
            <cstring>depth</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="depth">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Display Levels:</string>
           </property>
           <property name="buddy">
            <cstring>levels</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="levels">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="3">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Start at:</string>
           </property>
           <property name="buddy">
            <cstring>startValue</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="ListsSpinBox" name="startValue">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Alignment:</string>
          </property>
          <property name="buddy">
           <cstring>alignment</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="alignment"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="restartNumbering">
        <property name="text">
         <string>Restart numbering at this paragraph</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="letterSynchronization">
        <property name="text">
         <string>Letter Synchronization</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ListsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>dialogs/ListsSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
